test/rules/quotemark/double/test.ts.lint

```diff
@@ -8,6 +8,7 @@ var tabNewlineWithinSingle = 'tab\tNewline\nWithinSingle';
                              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~  [' should be "]
 'escaped\'quotemark';
 ~~~~~~~~~~~~~~~~~~~~ [' should be "]
+var unbalancedSingle = ('a)
```

Comment:
Adding this test case here means the rest of the file might not get linted properly, since following characters across lines will be considered part of the same (invalid) string. Plopping this file into astexplorer.net is showing that everything here is being considered a TaggedTemplateExpression.

Could you move this to a different file? Even if you moved this to the last line of this file, there's no guarantee later folks wouldn't add lines after it.

Reply:
That's what I assumed as well, until the tests actually pass regardless of where I list this example (at the start vs. here).

I can still move it to its own tests directory, just seemed like overkill for this one test case. Could name it something like `unbalanced-quotes`?

Reply:
Sounds good! Would you mind adding tests for `"` and <code>`</code as well? On second glance I think that's valuable to test.